odoo/exceptions.py

```diff
@@ -13,24 +13,31 @@
 import logging
 from inspect import currentframe
 from .tools.func import frame_codeinfo
+from .tools.translate import _
 
 _logger = logging.getLogger(__name__)
 
 
 # kept for backward compatibility
 class except_orm(Exception):
-    def __init__(self, name, value=None):
+    def __init__(self, name, value=None, action=None):
```

Comment:
What about having several possible actions?

Reply:
I had already implemented it locally, but pushed it just now